mage/main.go

```diff
@@ -591,6 +591,8 @@ func Compile(goos, goarch, ldflags, magePath, goCmd, compileTo string, gofiles [
 		gofiles[i] = filepath.Base(gofiles[i])
 	}
 	buildArgs := []string{"build", "-o", compileTo}
+	buildArgs = append(buildArgs, "-trimpath")
+
```

Comment:
Hey, thank you for this proposed change, what do you think about unifying these two lines since the append is not optional?

```suggestion
	buildArgs := []string{"build", "-o", compileTo, "-trimpath"}

```

Reply:
fine with me! Sorry what did I miss on why it's not optional?

Reply:
sorry, I chose a particular bad english for that sentence.

I meant that, since this will be always:
* Create string slice with 3 items
* Append one item

We could as well create a 4 item slice and save the resize, since it will always happen